homeassistant/components/renson/services.yaml

```diff
@@ -0,0 +1,143 @@
+set_timer_level:
+  fields:
+    timer_level:
+      required: true
+      advanced: false
+      default: "level1"
+      selector:
+        select:
+          translation_key: "fan_setting"
+          options:
+            - "level1"
+            - "level2"
+            - "level3"
+            - "level4"
+            - "holiday"
+            - "breeze"
+    time:
+      required: true
+      advanced: false
+      default: 0
+      selector:
+        number:
```

Comment:
There is a type mismatch between the schema and this file. `int` vs `time`
Please recheck all types

Reply:
Fixed mismatch. It should be int.